cve_bin_tool/checkers/bluez.py

```diff
@@ -6,13 +6,34 @@
 
 """
 from . import Checker
+from re import search
 
 
 class BluezChecker(Checker):
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS = [
+        r"Dialog Semiconductor GmbH",
+        r"Nordic Semiconductor ASA",
```

Comment:
I think these are likely to have false positives if these companies produce any other software, so they're likely not good patterns for this checker.

Reply:
> I think these are likely to have false positives if these companies produce any other software, so they're likely not good patterns for this checker.

I checked out. These companies only work on bluetooth adapters. I will look for something else if you say so

Reply:
Just because they only work on bluetooth adapters doesn't mean anything that mentions their names, though.  For example, if you search for https://github.com/search?q=nordic+semiconductor you get a lot of people's software for use with their products. It's possible that adding GmbH on there reduces the chances of a false positive, but it's only time.